### ISSUE TITLE
License protection

### ISSUE THREAD

**Issue by Viewline-MH** [armandomusto/Manhole-Report#47]
I am looking at a system that looks good for our purposes. I'm not sure where you have got with this but you don't have to do that part. We will arrange software protection. How are things generally, nearly there?

**Comment by armandomusto** [comment]
I just imported the code to check license from the old application, but still not tested.
I am waiting to close all open point and then complete with license system. 
Let me know what have you found.

**Comment by Viewline-MH** [comment]
https://www.microcosm.com/products/copyminder

We deal with these people already. Looking through it looks as if I can control licenses via the Shell route.

**Comment by armandomusto** [comment]
You can also manage licenses through a web user interface

**Comment by Viewline-MH** [comment]
Yes I like that. We know if the application is being used on more than one
machine and we can disable use. I am right in thinking that all I need from
you is a version that naturally would install and be available for
unlimited use and this system will enable to build rules around use?







*www.viewline.tv* <http://www.viewline.tv/>

t: (+44) 01752 358142
m. 07722 161 232
m. 07961 736 412

Viewline, Eurotech House, Burrington Ind. Est., Plymouth. PL5 3LZ

**Comment by armandomusto** [comment]
We need to integrate their libraries into your application to make the license working

**Comment by Viewline-MH** [comment]
Okay, I think that means something you can do ;)

**Comment by Viewline-MH** [comment]
On this I think we just need to get to a point where there is a clean, working .exe. I am pretty sure that the Copyminder process should enable shell control by me. if I can't get that working then we go down their library route.
